### ISSUE TITLE
Soy: Syntax highlighting error: {css ...}

### ISSUE THREAD

**Issue by DisDis** [vtst/ow#28]
Example 3 errors

Error message:"extraneous input '0' expecting '}'"

``` html
<div class="{css enable-0}">
</div>
<div class="{css enable-1}">
</div>
<div class="{css enable--1}">
</div>
```

Error message:"mismatched input 'in' expecting RULE_IDENT"
Error message:"mismatched input '}' expecting '{/template}'"

``` html
<div class="{css in-focus}">
</div>
<div class="{css out-focus}">
</div>
```

Error message:"extraneous input 'false' expecting '}'"

``` html
<div class="{css enable-false}">
</div>
<div class="{css enable-true}">
</div>
```

**Comment by DisDis** [comment]
SOY plugin v1.0.3
Eclipse 3.7.2

**Comment by vtst** [comment]
Thanks for the detailed bug report. I fixed the issue in the code base, it will be released with the next version, hopefully by the end of the week.

**Comment by vtst** [comment]
Included in version 1.0.4 released today.
